### PR TITLE
Document approval request workflow and API

### LINKED ISSUE #495: Document approval request workflow and API

### PR DESCRIPTION
**Pull request by rehandalal**
Fixes #495 

f?

### CODE REVIEW

File: recipe-server/normandy/settings.py [hunk 1]
```diff
@@ -277,6 +282,9 @@ class Development(Base):
 
     API_CACHE_TIME = values.IntegerValue(0)
 
+    SWAGGER_SETTINGS = Base.SWAGGER_SETTINGS
```
Comment (mythmon) [inline recipe-server/normandy/settings.py]: Did you mean `Core.SWAGGER_SETTINGS`? Also, I don't think you need this anyways, since it is a subclass.

Comment (rehandalal) [inline recipe-server/normandy/settings.py]: Oops yeah. I needed it to be able to make the next line happen.

Comment (rehandalal) [inline recipe-server/normandy/settings.py]: Although maybe there's a cleaner way of making that happen?

**Comment by rehandalal** [comment]
r?

**Pull request merged**